cogip/tools/detector/detector.py

```diff
@@ -179,28 +185,28 @@ def filter_distances(self) -> List[int]:
 
         # Iterate over all angles, starting by the first angle without obstacle
         first = start
-        while first < start + self.NB_ANGLES:
-            dist_min = self._lidar_data[first % self.NB_ANGLES]
+        while first < start + 360:
+            dist_min = self._lidar_data[first % 360]
 
             # Find the next angle with obstacle
             if dist_min >= self._max_distance:
                 first += 1
                 continue
 
             # An angle (first) with obstacle is found, iterate until the next obstacle without obstacle
```

Comment:
not in PR but it should be `iterate until the next angle without obstacle` ?

Reply:
Fixed.